### ISSUE TITLE
Logo Design Offer For Dart Time Machine

### ISSUE THREAD

**Issue by iqbalhood** [Dana-Ferguson/time_machine#1]
Hello Sir. I'm a UI/UX and Graphics Designer. who has contacted you via email last week.
Can I provide the logo contribution and make discussion about the logo on this issue?

**Comment by Dana-Ferguson** [comment]
Yeah, go right ahead.

:)

**Comment by iqbalhood** [comment]
Hello, Sir @Dana-Ferguson . These are my logo proposal for your application.
I'm very welcome if you have any comments, opinions, or revisions. Please just tell me.
I'll provide all file types and sizes you needed.
Hope you like it.

## Version 1
![dart time machine](https://user-images.githubusercontent.com/1563756/42860746-25e0e8f6-8a82-11e8-9c61-5ad36d298266.jpg)

## Version 2
![dart time machine logo 2](https://user-images.githubusercontent.com/1563756/42860745-25b26fb2-8a82-11e8-9396-8569882e6adb.jpg)

## Version 3
![dart time machine 3](https://user-images.githubusercontent.com/1563756/42860744-25807174-8a82-11e8-906e-d03824c767c2.jpg)

**Comment by Dana-Ferguson** [comment]
I took some time to think about and I solicited opinions from other people.
* People liked the arrow becoming a part of the page
* People were confused about the magnifying glass (myself included)
* Feedback from a visual designer
   * Inconsistent spacing between the arrow & page and clock hands & center circle
   * Clock hands not touching the center circle and the arrow touching the page

It's borderline busy, but looks safe.

### Icon
![mstile-150x150](https://user-images.githubusercontent.com/7284858/42921082-ea02b902-8ae7-11e8-84f9-fe467bb48b70.png)

### Website Favicon
![favicon-32x32](https://user-images.githubusercontent.com/7284858/42921039-be2ce8ca-8ae7-11e8-9b4e-11087d01f680.png)

Seems to maintain okay consistency at icon sizes. The magnifying glass gets pretty lost as a favicon (but, I think that's fine -- the favicon still looks unique).

I like the start so far, but... is there anything we can put into the space used by the magnifying glass \ page to better indicate that the project is global?

Thank you for your awesome work.

**Comment by iqbalhood** [comment]
Hello, Sir @Dana-Ferguson . These are improvement for your feedback. I add the globe symbol for indicate that the project is global. 

### Version 1

![1](https://user-images.githubusercontent.com/1563756/42980175-6b9bb36a-8c00-11e8-8f00-0523399d1593.jpg)


### Version 2
![4](https://user-images.githubusercontent.com/1563756/42980177-72f5bdf4-8c00-11e8-8da0-49e6dee7c479.jpg)

**Comment by Dana-Ferguson** [comment]
![sketch - tm logo](https://user-images.githubusercontent.com/7284858/43235070-433ab06e-904d-11e8-940c-47861d5d9bc5.png)

I'm thinking something like this. It combines the first version with the globiness of the current version.

**Comment by iqbalhood** [comment]
Okay, I Will Make it and Provide it soon..

**Comment by iqbalhood** [comment]
Hello Sir . These are my improvement for your last feedback. 
![whatsapp image 2018-08-03 at 16 54 28](https://user-images.githubusercontent.com/1563756/43763325-aecbf548-9a54-11e8-9588-8c3d994ca110.jpeg)

![whatsapp image 2018-08-03 at 16 54 37](https://user-images.githubusercontent.com/1563756/43763326-af016296-9a54-11e8-8db4-3127aadd485c.jpeg)

Hope you like it.

**Comment by Solido** [comment]
You've a concept here ! as a user I like it.
Do you thing a version with Capitalised letter with vertical inline would work ? 
 
Dart
Time
Machine

DTM lib !

When you complete the Logo I can animate it if you like.

**Comment by Dana-Ferguson** [comment]
Yeah, that is pretty good. I like it.

> Do you thing a version with Capitalised letter with vertical inline would work ?
>
> Dart
> Time
> Machine

It's so early in the morning for me. I'm interested in this, but I can't visualize it right now.

**Comment by Dana-Ferguson** [comment]
Cool, I went to add a logo to the front page Readme -- and, they're too big. I went to download and edit out the whitespace and noticed they're jpeg. They need to be png or svg (and preferably trimmed to size).

![image](https://user-images.githubusercontent.com/7284858/43938856-0f4c8afa-9c34-11e8-81e3-cf13f7740a4b.png)

And, can we get these edges to have consistent styling. I'm wondering if it would look better if we those edges  look more like the arrow embedded in the globe (5 pixel'ish spacing and conforming to the shape it's next to).

Thanks!
